slp2mp4/config.json

```diff
@@ -2,5 +2,6 @@
     "melee_iso": "~/Games/SSBM-v1.02-NTSC.iso",
     "dolphin_dir": "~/dolphin/Slippi-FM-r18/playback",
     "ffmpeg": "/usr/bin/ffmpeg",
-    "remove_short": true
+    "remove_short": true,
```

Comment:
imo both these should be command line options, and config.json should just have things that need to be set up once/infrequently.
However probably just leave it for this PR